doc/source/conf.py

```diff
@@ -133,7 +133,9 @@
     rst_epilog += f.read()
 
 
-linkcheck_ignore = ["https://sphinxdocs.ansys.com/version/*"]
+linkcheck_ignore = [
+    "https://sphinxdocs.ansys.com/version/*",
```

Comment:
If it's a regex you should take it in raw format 
```suggestion
    r"https://sphinxdocs.ansys.com/version/*",
```